linkml/generators/sqlddlgen.py

```diff
@@ -204,6 +205,11 @@ def _class_name_to_table(self, cn: ClassDefinitionName) -> str:
         """
         return underscore(cn)
 
+    def generate_header(self):
```

Comment:
are you sure '#' as a comment delimiter is respected by all SQL engines?

Currently in this module we only write DDL via sqlalchemy -- this should take care of emitting comments with the correct syntax

Reply:
Thank you for pointing that out @cmungall ! 

I made the change such that we are using the more traditional SQL comment syntax (`/* COMMENT */`)